test/test_image.py

```diff
@@ -48,3 +49,27 @@ def test_gradcheck(self):
         input = utils.tensor_to_gradcheck_var(input)  # to var
         assert gradcheck(image.gaussian_blur, (input, kernel_size, sigma,),
                          raise_exception=True)
+
+
+@pytest.mark.parametrize("window_size", [5])
+def test_get_laplacian_kernel(window_size):
```

Comment:
add some numeric unittest. Introduce manually input/expected values and use torch.testing assert_allclose

Reply:
I was meaning to have something like
```python
class TestGetLaplacianFilter:
    def test_get_filter_w3(...):
        pass
    def test_get_filter_w5(...):
        pass
```
consider add as needed

Reply:
This looks neat. What about the similar funcitons in tests for the gaussian. SHoudl they change too? Right now it is in the spirit similar to that.

Reply:
I'll explore more tests functions available in PyTorch in due time.